sdk/Lusid.Sdk.Tests/tutorials/Misc/ComplexInstruments.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Collections.Generic;
+using Lusid.Sdk.Model;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Misc
```

Comment:
I think we should put these in a new Tutorials/Instruments folder so its easy for users to find

Reply:
ok